app/models/question/select_all_that_apply.rb

```diff
@@ -7,6 +7,7 @@
 # duplication than more complicated inheritance.
 class Question::SelectAllThatApply < Question
   self.type_name = "Select All That Apply"
+  self.model_exporter = 'traditional_type'
```

Comment:
doesn't traditional refer to multiple choice? not SATA? Will it use the same exporter?

Reply:
the export mapping logic is the same... this model_exporter translates to the method name in the export formatters. You can see the various types in the sample files we attached.